src/lib/youTubeApi.js

```diff
@@ -0,0 +1,17 @@
+const key = 'AIzaSyCbnwRIBvWCoQGaLIOFUx4rhLHg-Sb1a8s';
```

Comment:
😱 
Do not save your api key at public repositories, it is better if you handle it with environment variables. Check [dotenv](https://www.npmjs.com/package/dotenv) for example.